assets/js/components/notifications/UserInputSettings.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * UserInputSettings component.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId as useInstanceID } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Notification from './notification';
+import { getTimeInSeconds } from '../../util';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import personSitImage from '../../../images/person-sit.png';
```

Comment:
This wasn't specifically mentioned in the ACs, but could we use an SVG for this image? I think it makes sense going forward to use SVGs for any illustration (unless we don't have one), because they are vector images and thus "endlessly scalable", plus they're usually smaller files. Figma allows to export anything as SVG, so that should be trivial.

We will need to add support to the `Nofication` component so that it can receive an image as SVG - maybe through a different prop like `ImageSVG`?

Here's the SVG of the person sitting (as exported from Figma):
```html
<svg width="571" height="178" viewBox="0 0 571 178" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0)">
<path d="M370.343 115.333C368.47 114.112 366.645 112.81 364.876 111.429C350.543 100.235 343.209 82.6431 330.889 69.5162C318.831 56.671 304.981 44.2139 290.173 34.6695C259.135 14.6628 222.658 7.46021 189.01 16.9038C153.062 26.9949 122.14 53.8814 106.5 87.9088C90.856 121.936 90.5897 162.441 105.215 196.569C118.61 227.829 145.616 257.417 178.204 268.445C202.999 276.836 229.225 274.006 253.851 269.121C287.935 262.36 320.157 255.315 356.016 259.44C378.141 261.985 398.575 266.721 419.803 260.612C439.531 254.934 456.127 238.79 465.717 220.994C475.029 203.714 478.183 182.546 473.038 162.457C469.147 147.254 458.973 131.649 443.871 127.413C433.867 124.608 423.709 127.251 413.593 127.672C398.704 128.295 383.358 123.806 370.343 115.333Z" fill="#F8F9FA"/>
</g>
<g clip-path="url(#clip1)">
<path fill-rule="evenodd" clip-rule="evenodd" d="M204.25 123.277C204.25 123.277 219.273 142.228 223.224 143.768C227.175 145.308 269.853 144.815 272.97 144.45C276.088 144.085 275.519 146.464 278.338 146.615C281.257 146.776 282.939 144.194 283.336 142.891C285.118 144.872 288.259 147.918 290.32 146.857C291.779 146.099 291.798 145.435 290.794 144.388C292.215 145.208 294.783 146.411 295.82 145.601C297.488 144.303 297.644 143.09 297.081 142.081C295.342 138.959 289.742 134.539 288.411 134.027C286.876 133.421 282.199 135.088 279.949 135.226C269.749 135.856 244.028 129.512 235.59 129.128C234.348 126.759 221.485 111.205 220.49 109.651C216.961 112.669 204.25 123.277 204.25 123.277Z" fill="#F3B091"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M271.734 142.906C271.734 139.684 276.803 137.078 283.052 137.078C289.301 137.078 294.371 139.684 294.371 142.906C294.371 146.127 289.301 148.728 283.052 148.728C276.803 148.728 271.734 146.132 271.734 142.906Z" fill="#AECBFA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M216.221 143.793H432.962C434.119 143.793 435.056 144.731 435.056 145.888V149.678C435.056 150.834 434.119 151.772 432.962 151.772H216.221C215.064 151.772 214.126 150.834 214.126 149.678V145.888C214.126 144.731 215.064 143.793 216.221 143.793Z" fill="#D2E3FC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M171.507 165.083C202.876 156.247 247.875 161.141 257.582 174.62C261.05 179.433 259.875 186.966 259.004 189.804C258.132 192.642 221.424 271.397 221.424 271.397L204.965 265.139L221.14 201.18C221.14 201.18 167.608 166.172 171.507 165.083Z" fill="#BDC1C6"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M398.462 147.08H404.69C405.717 147.08 406.55 148.902 406.55 151.149V301.724C406.55 303.972 405.717 305.794 404.69 305.794H398.462C397.434 305.794 396.601 303.972 396.601 301.724V151.149C396.601 148.902 397.434 147.08 398.462 147.08Z" fill="#D2E3FC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M148.495 146.544C119.798 206.486 177.717 208.102 198.042 207.803C218.367 207.505 223.716 205.974 229.216 204.643C230.974 209.968 252.043 278.931 252.043 278.931L270.544 273.34C270.544 273.34 256.401 185.73 253.208 179.813C244.439 163.543 192.565 162.662 186.003 163.055C179.442 163.449 152.133 138.945 148.495 146.544Z" fill="#DADCE0"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M183.248 53.5942C183.248 49.6233 185.64 46.0436 189.309 44.5244C192.977 43.0053 197.2 43.846 200.007 46.6545C202.814 49.463 203.653 53.6861 202.132 57.3541C200.611 61.0222 197.03 63.4127 193.059 63.4108C190.456 63.4108 187.96 62.3764 186.12 60.5353C184.279 58.6942 183.246 56.1973 183.248 53.5942Z" fill="#B83428"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M184.407 72.6592C152.934 72.6971 142.416 146.568 145.704 151.632C148.116 155.347 176.665 166.267 186.449 162.913C189.902 161.729 198.89 143.275 203.978 124.656C209.403 120.202 219.67 111.769 221.342 110.144C220.868 106.577 212.62 92.2023 204.078 85.8348C201.126 79.093 195.649 72.6592 184.407 72.6592Z" fill="#FCC934"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M192.546 122.85C192.546 122.85 207.57 141.801 211.521 143.341C215.472 144.881 258.15 144.388 261.267 144.023C264.385 143.658 263.816 146.037 266.635 146.188C269.553 146.349 271.235 143.767 271.633 142.465C273.415 144.445 276.556 147.491 278.617 146.43C280.076 145.672 280.095 145.009 279.091 143.962C280.512 144.781 283.08 145.985 284.117 145.175C285.785 143.876 285.941 142.664 285.378 141.654C283.639 138.532 278.039 134.112 276.707 133.6C275.172 132.994 270.496 134.661 268.246 134.799C258.046 135.429 232.324 129.085 223.886 128.701C222.645 126.333 209.782 110.779 208.787 109.225C205.258 112.243 192.546 122.85 192.546 122.85Z" fill="#F3B091"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M188.353 57.4653C188.351 51.4359 191.982 45.9993 197.552 43.6911C203.122 41.3829 209.534 42.6578 213.797 46.9212C218.061 51.1845 219.336 57.5966 217.027 63.1666C214.719 68.7366 209.283 72.3673 203.253 72.3654C195.025 72.3628 188.356 65.6933 188.353 57.4653Z" fill="#F3B091"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M190.354 54.2197C190.415 56.5365 185.564 71.1855 183.65 73.9903C184.597 76.2312 193.869 81.8502 198.128 80.6515C198.602 77.8089 200.189 72.0952 201.17 70.8255C202.151 69.5558 190.292 51.884 190.354 54.2197Z" fill="#F3B091"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M207.385 26.0493C204.519 26.0581 201.801 27.3262 199.952 29.5174C198.727 29.0553 197.429 28.8146 196.119 28.8067C192.558 28.756 189.189 30.4153 187.059 33.2688C184.928 36.1222 184.295 39.8241 185.355 43.2236C184.017 44.4551 183.254 46.19 183.251 48.0087C183.273 51.2018 185.56 53.9294 188.7 54.5089C188.776 54.1441 188.776 53.8883 188.894 53.4713C189.216 52.3627 194.039 50.335 194.579 50.6998C195.005 51.0048 195.489 51.2178 196.001 51.3251C196.045 51.3277 196.089 51.3277 196.133 51.3251C200.212 51.3147 203.965 49.0929 205.936 45.5214C206.419 45.6064 206.909 45.6555 207.4 45.6683C207.4 51.0898 211.795 55.4849 217.216 55.4849C222.638 55.4849 227.033 51.0898 227.033 45.6683C227.033 40.2467 222.638 35.8517 217.216 35.8517C217.213 33.2482 216.175 30.7528 214.331 28.9145C212.487 27.0762 209.989 26.0456 207.385 26.0493Z" fill="#B83428"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M184.404 48.4351C184.412 45.2499 186.999 42.6735 190.184 42.6787C193.369 42.684 195.948 45.2689 195.945 48.454C195.942 51.6391 193.36 54.2198 190.174 54.2198C188.641 54.2198 187.172 53.6099 186.089 52.5247C185.006 51.4394 184.4 49.968 184.404 48.4351Z" fill="#F3B091"/>
<path d="M211.157 57.8584C211.01 59.1708 211.299 62.1224 212.289 63.411" stroke="#B83428" stroke-linecap="round" stroke-linejoin="round"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M203.168 56.1151C203.642 54.9923 204.59 54.3148 205.248 54.6038C205.907 54.8928 206.039 56.0251 205.551 57.1574C205.063 58.2897 204.13 58.9578 203.472 58.6688C202.813 58.3798 202.676 57.2427 203.168 56.1151Z" fill="#B83428"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M214.42 58.678C214.894 57.5552 215.842 56.8777 216.5 57.162C217.159 57.4462 217.291 58.5833 216.803 59.7156C216.315 60.8479 215.382 61.5159 214.724 61.2269C214.065 60.9379 213.928 59.7961 214.42 58.678Z" fill="#B83428"/>
<path d="M203.113 66.9218C201.254 66.3319 199.929 64.6857 199.75 62.7432" stroke="#B83428" stroke-linecap="round" stroke-linejoin="round"/>
<path d="M239.782 131.108L224.019 128.702L214.236 116.137" stroke="#B83428" stroke-linecap="round" stroke-linejoin="round"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M290.95 64.8325H358.884C361.812 64.8325 364.186 67.2061 364.186 70.134V126.234C364.186 129.161 361.812 131.535 358.884 131.535H290.95C288.026 131.53 285.658 129.158 285.658 126.234V70.134C285.658 67.2098 288.026 64.8377 290.95 64.8325Z" fill="#D2E3FC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M315.501 114.996H338.437L351.281 144.208H327.535L315.501 114.996Z" fill="#AECBFA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M244.485 147.08H250.713C251.741 147.08 252.574 148.907 252.574 151.161V302.186C252.574 304.44 251.741 306.267 250.713 306.267H244.485C243.458 306.267 242.625 304.44 242.625 302.186V151.161C242.625 148.907 243.458 147.08 244.485 147.08Z" fill="#D2E3FC"/>
</g>
<defs>
<clipPath id="clip0">
<rect width="381" height="261" fill="white" transform="translate(94.4995 12.7935)"/>
</clipPath>
<clipPath id="clip1">
<rect width="298" height="280" fill="white" transform="translate(136.5 25.7935)"/>
</clipPath>
</defs>
</svg>
```

Super-nitpick, but let's call it `person-sitting` which is a bit more grammatically correct than `person-sit`.

Reply:
Related: https://github.com/google/site-kit-wp/pull/2397/files#r529025540

Reply:
> Super-nitpick, but let's call it `person-sitting` which is a bit more grammatically correct than `person-sit`.

I saw this too, but to be fair it is named like this in Figma. 👍  from me though.

---

I chimed in on the related discussion as well.